server/handlers/messageHandler.js

```diff
@@ -17,7 +17,7 @@ const bulkMessagePostSchema = Joi.object({
   author_handle: Joi.string().required(),
   subject: Joi.string().required(),
   body: Joi.string(),
-  type: Joi.string().required(),
+  type: Joi.string().required().valid('announce', 'survey'),
```

Comment:
can there be a type 'message' for /bulk_message POST requests? @ZavenArra

Reply:
no